config/300-pipelinerun.yaml

```diff
@@ -26,20 +26,20 @@ spec:
     - tekton-pipelines
   scope: Namespaced
   additionalPrinterColumns:
-    - name: Type
+    - JSONPath: .status.conditions[*].type
```

Comment:
We should remove this one and just have the next column `Status` to be `Succeeded` :angel: